docker-compose.yaml

```diff
@@ -63,4 +63,6 @@ services:
     environment:
       - NOTEBOOK_PORT=4041
       - SPARK_MASTER_URL=spark://spark-master:7077
-      - SPARK_DRIVER_HOST=spark-notebook
\ No newline at end of file
+      - SPARK_DRIVER_HOST=spark-notebook
+    volumes:
+      - ./cdm_shared_workspace:/cdm_shared_workspace
```

Comment:
Where are you mounting from in rancher?

Reply:
in Rancher I just do `/cdm_shared_workspace:/cdm_shared_workspace`.

Reply:
Where does the data live though? In the VM the notebook is running on?

Reply:
I think so. Double confirming with devops team in slack.